### ISSUE TITLE
Unable to find csc.exe

### ISSUE THREAD

**Issue by theonlygusti** [OrangeT/vim-csharp#37]
I'm using macOS Sierra and mono for compiling and running my C# code.

Whenever I start vim, I get the message `Unable to find csc.exe` before vim actually starts up. This is kinda frustrating.

**Comment by jorgeflh** [comment]
Yes, same is happening here. I'm using in Debian 8.

**Comment by msawangwan** [comment]
same. centOS 7

**Comment by rmboggs** [comment]
Yup, same here on OpenBSD.

Looking at the code, the screenshots, and the some of the notes, I think this plugin is specific to .NET on Windows.  This plugin may not support Mono on a non-Windows system.

It looks like the offending code is in the autoload/cs.vim file if anyone has experience with vim scrips and has the time.  Not sure this project is active anymore.

**Comment by kianryan** [comment]
Hello.  Apologies - I've not looked at this code for quite a while.  I'll try and find some time in the next couple of weeks.

If people come up with pull requests, and they look sensible, I'm happy to merge them in.

**Comment by rmboggs** [comment]
No worries.  Thank you for taking a look.

Am I right in thinking that the current state of this plugin is specific to .NET on Windows systems?

**Comment by kianryan** [comment]
You are correct.  And the last revision was for .NET 4.5.  There are some wobbles with newer framework revisions that I need to look in to.

**Comment by kianryan** [comment]
Okay, please try latest version.

Last few commits should resolve this.

**Comment by rmboggs** [comment]
Tried it, it's complaining of an undefined variable name: 'msbuild', looks like in the cs#get_ net_compiler function, line 21 I think it says.

**Comment by kianryan** [comment]
Try that please - apparently I can't type. ;-)

**Comment by rmboggs** [comment]
Seems to be working better so far.  I need to test it more though.  I'll let you know what I find.

Thanks for taking the time to address.
